fontbe/src/features/marks.rs

```diff
@@ -621,6 +627,62 @@ fn resolve_anchor_once(
     Ok(anchor)
 }
 
+fn get_ligature_carets(
```

Comment:
Is there a python equivalent to link to?

I suggest we should have a comment somewhere, perhaps here, that captures, or links to, the discussion of whether we need the table, why / who uses it.